### PR TITLE
add pandera api doc

### PR DESCRIPTION
**Pull request by smackesey**
### Summary & Motivation

Adds API doc page for Pandera

### How I Tested These Changes

Vercel preview

### CODE REVIEW

**Comment by smackesey** [comment]
Current dependencies on/for this PR:
* master
  * **PR #9246** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/9246" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/9246?utm_source=stack-comment).

File: docs/content/_navigation.json [hunk 1]
```diff
@@ -711,6 +711,10 @@
             "title": "Pandas (dagster-pandas)",
             "path": "/_apidocs/libraries/dagster-pandas"
           },
+          {
+            "title": "Pandera (dagster-pandas)",
```
Comment (erinkcochran87) [inline docs/content/_navigation.json:715]: Is this copy pasta? Should it be `dagster-pandera` here and in `path`?

**Review by erinkcochran87 — COMMENTED** [review]
Left a comment on what I think is copypasta.

This also isn't loading for me in Vercel at `/_apidocs/libraries/dagster-pandera`, so I'm not sure if there's something else going on.

**Review by yuhan — CHANGES_REQUESTED** [review]
overall looking good - could you update the PR to isolate the changes from the earlier stack?

File: docs/content/_navigation.json (hunk 1 above)
Comment (smackesey) [reply docs/content/_navigation.json:1111]: yes

**Comment by smackesey** [comment]
@erinkcochran87 with yuhan out, could you have a look? I addressed her request.

**Review by erinkcochran87 — APPROVED** [review]
lgtm - thanks Sean!

**Pull request merged**